src/napari_plugin_manager/base_qt_package_installer.py

```diff
@@ -478,6 +478,7 @@ def cancel_all(self):
                 process.errorOccurred.disconnect(self._on_error_occurred)
 
             self._end_process(process)
+            process.waitForFinished()
```

Comment:
Should this line be part of `_end_process()`? There are more calls to this method that don't wait 🤔